### DISCUSSION TITLE
🟠 InjectionToken | Angular Challenges

### DISCUSSION THREAD

**Discussion by giscus** [tomalaforge/angular-challenges#702] [Announcements]
# 🟠 InjectionToken | Angular Challenges

Challenge 39 is about learning the power of dependancy injection

https://angular-challenges.vercel.app/challenges/angular/39-injection-token/

**Comment by svenson95** [comment]
I really like the scenario in this challenge, "The development team decided to use a global constant ..." and "However, a few weeks later, the product team wants to add a new screen ...". This makes the challenge much more enjoyable imo :-)
